### PR TITLE
Transform decorators for @connect

### PR DESCRIPTION
**Pull request by davidroman0O**
Hi !

A little contribution with a babel plugin called "transform-decorator" who help me a lot in several projects. 
I've added some example and hope you'll like it! 😄 

Thx for the starter kit ;)

### CODE REVIEW

**Comment by LeoLeBras** [comment]
Hey @warka0,

Decorators are experimental feature, you don't need decorators or ES7 to use `connect()`. Take a look at this issue [Don't use decorators](https://github.com/happypoulp/redux-tutorial/issues/87). 

You can use transform-decorator by using this as your .babelrc instead :

```
{
  "presets": ["react-native-stage-0/decorator-support"]
}
```

**Comment by davidroman0O** [comment]
Hi,

Just a quick note about decorator. You're right, it sucks in production !
tldr: too much different kind of errors

Thx for the advice ;)
